eth/stagedsync/stage_headers.go

```diff
@@ -292,6 +305,17 @@ func HeadersUnwind(u *UnwindState, s *StageState, tx kv.RwTx, cfg HeadersCfg, te
 		var maxTd big.Int
 		var maxHash common.Hash
 		var maxNum uint64 = 0
+		// unwind the merge
+		isTrans, err := rawdb.Transitioned(tx, u.UnwindPoint)
+		if err != nil {
+			return err
+		}
+
+		if cfg.chainConfig.TerminalTotalDifficulty != nil && !isTrans {
+			if err := tx.Delete(kv.HeaderTD, dbutils.HeaderKey(math.MaxUint64, common.Hash{}), nil); err != nil {
```

Comment:
Why is this still here?

Reply:
you mean the chainConfig? it is still there because in the header config, chain config is needed for others things other thank ttd, it is not something i added myself